Gemfile

```diff
@@ -3,9 +3,6 @@ source "https://rubygems.org"
 # Specify your gem"s dependencies in kitchen-ec2.gemspec
 gemspec
 
-gem "winrm-transport"
-gem "winrm-fs"
```

Comment:
👍 I had done this locally while troubleshooting #431 yesterday, but hadn't hunted down whether they were really needed.

Reply:
Yeah - I didn't see anywhere they were used, and `winrm-transport` has been completely deprecated

Reply:
![buh-bye deprecated code](https://user-images.githubusercontent.com/517302/49185705-8dee8900-f330-11e8-8db4-7a4ee7aac43e.gif)